.travis.yml

```diff
@@ -33,7 +33,9 @@ script:
   - flake8 --config=.flake8 .
   - mypy cgp
   - if [ "$DEP" = "[all]" ]; then
-       cd docs && make html && cd ../ || exit 1;
+       make -C docs/ html || exit 1;
+       touch docs/build/html/.nojekyll
```

Comment:
is the difference between `build` and `_build` (used below) on purpose or a bug?

Reply:
oops, no, this is wrong.

Reply:
fixed, b5462067211cef93dcd68d602b6514aac917bb29